packages/components/pie-webc-core/src/functions/dispatchCustomEvent.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Dispatch a custom event.
+ *
+ * To be used whenever we have behavioural events we want to
+ * bubble up through the component.
+ * @param {Element} element The element to dispatch the event from.
+ * @param {string} eventType
+ * @param {any} detail
+ */
+export function dispatchCustomEvent (element: Element, eventType: string, detail?: CustomEventInit['detail']) : void {
+    const event = new CustomEvent(eventType, {
+        bubbles: true,
+        composed: true,
+        detail,
```

Comment:
[For reference](https://github.com/justeattakeaway/pie/tree/main/packages/components#events)

Reply:
I commented on your initial review about bubbles and composed [here](https://github.com/justeattakeaway/pie/pull/1272#discussion_r1489077798). I think that as we are starting small and all the current use cases need both key values to be true we can leave it as is. If in future there will be the need to set either of them to false, the helper can be modified leaving the default as true to not break existing use cases. What do you think?

Good point about event name validation, I will add it in

Reply:
Ah, sorry I missed that! 

Yep, that sounds fine to me 👍 

It might be worth mentioning in the JSdoc that bubbles and composed will be true for now.